submissions/Dovahkiin1991/html-css-popup/css/styles.css

```diff
@@ -0,0 +1,181 @@
+html {
+    min-height:100%;
+    position:relative;
+}
+
+body {
+    background: transparent url('../images/wallpaper.jpg') no-repeat center top;
+    background-size: cover;
+    height: 100%;
+    font-family: sans-serif;
+}
+
+*:focus {
+    outline: none;
+    border-radius: 1px;
+    box-shadow: 0 0 0 2px #84a6bc;
+}
+
+.header {
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+.header--sticky {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    background: rgba(255, 255, 255, 0.4);
+    backdrop-filter: blur(50px);
+    -webkit-backdrop-filter: blur(50px);
+    box-shadow: 0px 1px 6px 2px rgb(30 30 30 / 10%);
+}
+
+.navbar{
+    margin: 0 -10px;
+    padding: 0 2px;
+    color: #000;
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: flex-start;
+}
+
+.navbar input {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 0;
+    height: 0;
+    padding: 0;
+    margin: 0;
+    border: none;
+    -webkit-appearance: none;
+    appearance: none;
+    box-sizing: border-box;
+}
+
+.navbar input:focus {
+    box-shadow: none;
+}
+
+.navbar input:focus + label {
+    outline: none;
+    border-radius: 1px;
+    box-shadow: 0 0 0 2px #84a6bc;
+}
+
+.navbar__sub {
+    position: absolute;
+    top: 100%;
+    left: 0;
+    margin-left: 0;
+    margin-right: 0;
+    background: rgba(255, 255, 255, 0.4);
+    backdrop-filter: blur(50px);
+    -webkit-backdrop-filter: blur(50px);
+    opacity: 0;
+    visibility: hidden;
+    min-width: 150px;
+}
+
+.navbar__sub--icons {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.navbar__sub--icons .navbar__item {
+    flex: 0 0 33.333333%;
+}
+
+.navbar__sub--icons .navbar__item--full-width {
+    flex: 0 0 100%;
+    text-align: center;
+}
+
+.navbar__sub--icons .navbar__item--full-width:hover {
+    text-decoration: underline;
+    border-color: transparent;
+}
+
+.navbar__sub--icons .navbar__link {
+    border: 1px solid transparent;
+}
+
+.navbar__sub--icons .navbar__link:hover,
+.navbar__sub--icons .navbar__link:focus {
+    border-color: #000;
+}
+
+.navbar__item input:checked ~ .navbar__sub {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid too specific CSS selectors. [Read about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)